lib/scopes.js

```diff
@@ -0,0 +1,75 @@
+const { trackedIssuesQuery, updateIssueMutation } = require('./queries')
+
+const parseIssueDescription = async(context) => {
+    const regexSimple = /(Related\sto\s+(\S+)?#(\d+))/
```

Comment:
This won't match URLs, just `#ISSUE_NUMBER`, which is great but sometimes the Pitch or Bet is in another repo so we need to be able to understand full URLs here too.